dadab/dadab.js

```diff
@@ -3,77 +3,101 @@ based on:
  - Simple Image Glitching Processing - datamoshing.com | 2016
  - a pixel sorting sketch from Jerome Martinez | 2017
  */
- 
+
 // output needs to be 1066 x 800
- 
+
+var capture;
+var gif;
 var img;
-var imgFileName = "vera";
-var fileType = "jpg";
+var imgFileName = 'vera';
+var fileType = 'jpg';
+
+var START_FRAME = 15;
+var END_FRAME = 115;
+
+const serverUrl = 'http://localhost:3000';
+
+const socketOptions = {
+  transports: ['websocket'],
+  'force new connection': true
+};
+const socket = io(serverUrl, socketOptions);
 
-function preload() {
+function preload () {
   // load an image from our library to set the canvas createCanvas
-  img = loadImage(imgFileName + "." + fileType);
+  img = loadImage(imgFileName + '.' + fileType); // eslint-disable-line no-undef
```

Comment:
All of the `// eslint-disable-line no-undef` throughout here is to get my editor to stop yelling at me. It only half worked. 😃